File: programme/minimum-of-numbers/min.cs
// Write a programmeto print the minimum of given numbers
using System;
using System.Linq;

public class MinNumber
{
    static void Main(string[] args)
    {
        int Minimum(string input)
        {
            return input.Split(" ").Min(x => int.Parse(x));
        }

        // Input numbers
        Console.Write("Enter numbers : ");
        string input = Console.ReadLine();

        // output minimum number
        Console.WriteLine("Minimum : " + Minimum(input));
    }
}

File: programme/minimum-of-numbers/min.js
// Write a programmeto print the minimum of given numbers

"use strict";

// perform following command to run programme
// node min.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", (nums) => {
  nums = nums.split(" ").map(Number);
  console.log(`Output : ${Math.min(...nums)}`);
  rl.close();
});
